src/main/webapp/app/exercises/shared/result/result-detail.scss

```diff
@@ -1,93 +1,80 @@
-.feedback-item {
-    border-radius: 4px;
-    padding: 5px 10px;
-    margin-bottom: 15px;
-    font-weight: normal;
+:host {
+    $feedback-header-fg: #fff;
```

Comment:
Will look into that in the follow up. This is the same way how non-changing color is implemented in e.g. the artemis header